src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebResponseObjectFactory.CoreClr.cs

```diff
@@ -11,15 +11,11 @@ internal static class WebResponseObjectFactory
     {
         internal static WebResponseObject GetResponseObject(HttpResponseMessage response, Stream responseStream, ExecutionContext executionContext)
         {
-            WebResponseObject output;
-            if (WebResponseHelper.IsText(response))
+            WebResponseObject output = WebResponseHelper.IsText(response) switch
             {
-                output = new BasicHtmlWebResponseObject(response, responseStream);
-            }
-            else
-            {
-                output = new WebResponseObject(response, responseStream);
-            }
+                true => new BasicHtmlWebResponseObject(response, responseStream),
+                _ => new WebResponseObject(response, responseStream)
+            };
```

Comment:
Why not `WebResponseHelper.IsText(response) ? new BasicHtmlWebResponseObject(response, responseStream) : new new WebResponseObject(response, responseStream);`?

Reply:
done